packages/maskbook/src/web3/hooks/useBlockie.ts

```diff
@@ -1,17 +1,16 @@
 import { BlockieOptions, create } from 'ethereum-blockies'
 import { useMemo } from 'react'
 
-export function useBlockie(address: string, options?: BlockieOptions) {
+export function useBlockie(address: string, options_?: BlockieOptions) {
     return useMemo(() => {
-        const defaultOptions = {
+        const options = {
             seed: address,
             color: '#dfe',
             bgcolor: '#aaa',
         }
-        if (options) Object.assign(defaultOptions, options)
-
+        if (options_) Object.assign(options, options_)
```

Comment:
see https://lodash.com/docs/4.17.15#defaults

Reply:
I used {...} instead of loadas if it is ok.